### ISSUE TITLE
What does the warning flag in testing for maml mean? what does it mean to "fine tune for testing"?

### ISSUE THREAD

**Issue by brando90** [facebookresearch/higher#120]
Since maml already has it's fine-tuning step (it's adaptation step) I'm unsure what this means.

Does this mean that people change the meta-learned weights (slow weights) on the validation set again? Isn't that just making the data set larger in overall effect? Or something else?

```
    # Crucially in our testing procedure here, we do *not* fine-tune
    # the model during testing for simplicity.
    # Most research papers using MAML for this task do an extra
    # stage of fine-tuning here that should be added if you are
    # adapting this code for research.
```

https://github.com/facebookresearch/higher/blob/main/examples/maml-omniglot.py

cross posted: https://stats.stackexchange.com/questions/550990/what-does-it-mean-to-fine-tune-s-maml-model-for-testing

**Comment by brando90** [comment]
```
    # Crucially in our testing procedure here, we do *not* fine-tune
    # the model during testing for simplicity.
    # Most research papers using MAML for this task do an extra
    # stage of fine-tuning here that should be added if you are
    # adapting this code for research.
```
